support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -103,25 +62,4 @@ export const tests: Tests = {
 		seed: 2,
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 	},
-	makeItAnnualNudgeGlobal: {
-		variants: [
-			{
-				id: 'control',
-			},
-			{
-				id: 'variant',
-			},
-		],
-		isActive: true,
-		audiences: {
-			ALL: {
-				offset: 0,
-				size: 1,
-			},
-		},
-		omitCountries: countriesAffectedByVATStatus,
```

Comment:
If your in the test, you cant be in these countries, have I got that right. Now the test is removed (ie on) for the full audience, where do hide for `countriesAffectedByVATStatus`?